meta-integrity/classes/sign_rpm_ext.bbclass

```diff
@@ -38,4 +38,10 @@ python () {
         gpg_path = d.getVar('TMPDIR', True) + '/.gnupg'
         d.setVar('GPG_PATH', gpg_path)
 
+    if len(gpg_path) > 80:
+        msg =  "The default GPG_PATH '%s'\n" % gpg_path
+        msg += "of %d characters is too long.\n" % len(gpg_path)
+        msg += "Due to GPG homedir path length limit, please set GPG_PATH shorter than 80 characters"
+        raise bb.parse.SkipRecipe(msg)
```

Comment:
Could you use this format to generate the message?
```
msg = "xxxx %s xxxx".format(gig_path, len(gpg_path))
```

Additionally, where is the source info about the limit of path length of GPG_PATH?

Reply:
OK, no problem.
https://unix.stackexchange.com/questions/367008/why-is-socket-path-length-limited-to-a-hundred-chars
show the limit information